### PR TITLE
Remove python wrapper

### PR DESCRIPTION
**Pull request by duga3**

### CODE REVIEW

File: src/menu.cpp [hunk 1]
```diff
@@ -219,33 +217,6 @@ namespace App
...
                     redraw = true;
 
-                    PyObject *m, *d, *v;
-                    m = PyImport_AddModule("__main__");
-
-                    if (m == NULL)
-                        break;
-
-                    d = PyModule_GetDict(m);
-                    v = PyRun_String(commands.back().c_str(), Py_single_input, d, d);
```
Comment (rasmusgo) [inline src/menu.cpp:229]: This means that the console doesn't work anymore. Please hook up lua or angelscript!

File: thirdparty/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+angelscript
```
Comment (rasmusgo) [inline thirdparty/.gitignore:1]: angelscript is checked in so it should not be ignored completely! Perhaps you want to ignore `libangelscript.a`?

**Review by rasmusgo — CHANGES_REQUESTED** [review]

File: thirdparty/.gitignore (hunk 2 above)
Comment (duga3) [reply thirdparty/.gitignore:1]: Can we just remove anglescript?

Comment (rasmusgo) [reply thirdparty/.gitignore:1]: Yes, that would also work.

Comment (duga3) [reply thirdparty/.gitignore:1]: Guess one script lang binding does the job for a game that doesn't even work. :D

**Pull request closed without merging**